CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ SET( ${PROJECT_NAME}_Variant "Full" ) # the particular variant of CaPTk (Full/Ne
 
 SET( PROJECT_VERSION_MAJOR 1 )
 SET( PROJECT_VERSION_MINOR 8 )
-SET( PROJECT_VERSION_PATCH 0.nonRelease )
+SET( PROJECT_VERSION_PATCH 0.Beta )
```

Comment:
Could you change this to `nonRelease` as we are not tagging this?

Reply:
Done.